src/Compilers/CSharp/Test/Symbol/Compilation/ReferenceManagerTests.cs

```diff
@@ -72,30 +72,11 @@ public void VersionUnification_SymbolUsed()
             // reference asks for a higher version than available:
             var testRefV2 = CreateCompilation("public class E : D { }", new MetadataReference[] { new CSharpCompilationReference(refV2), v1 }, assemblyName: "testRefV2");
 
-            // TODO (tomat): we should display paths rather than names "refV1" and "C"
-
-            testRefV1.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=22&length=17)

It doesn't feel like simply removing the verification of diagnostics is appropriate. I think at least we should verify that there is no diagnostics.